### ISSUE TITLE
nukie war ops should start an auto-FTL timer for the ship

### ISSUE THREAD

**Issue by retequizzle** [space-wizards/space-station-14#22152]
keeps people from calling war ops then taking forever, something like 15-20 mins at most

**Comment by Brandon-Huu** [comment]
Horrible Idea *closes issue*

**Comment by Brandon-Huu** [comment]
Coordinating a plan with 5 people takes time.
If crew are bored they should do their jobs instead of trying to valid hunt

**Comment by deltanedas** [comment]
warops validhunting real

**Comment by retequizzle** [comment]
> Coordinating a plan with 5 people takes time.

maybe don't call war ops until people have agreed on doing it that way

**Comment by Brandon-Huu** [comment]
> > Coordinating a plan with 5 people takes time.
> 
> maybe don't call war ops until people have agreed on doing it that way

The only person who has to agree on it is the commander

**Comment by retequizzle** [comment]
> > > Coordinating a plan with 5 people takes time.
> > 
> > 
> > maybe don't call war ops until people have agreed on doing it that way
> 
> The only person who has to agree on it is the commander

more or less proves my point of "don't call it until you're ready then"

**Comment by deltanedas** [comment]
you only have like 5 mins to call it after that you cant

**Comment by retequizzle** [comment]
hence why i'm saying 15-20 mins beyond that before FTL kicks in, which with all things considered is a pretty fair assessment since we try to discourage major station damage early on in rounds during other game modes

could just make it uniform 30 minutes for nukie rounds in general before the shuttle auto-FTLs if it hasn't already, if you want more consistency

**Comment by Ratonero500** [comment]
This would make it possible to metagame nukies arrival since crew will know how much time it has to pass after war has been called,and has other have said nukies shouldnt have a time limit to plan,remember that they are 5 random people and so dont know each other,thus they need time agree and plan.

**Comment by retequizzle** [comment]
> This would make it possible to metagame nukies arrival since crew will know how much time it has to pass after war has been called,and has other have said nukies shouldnt have a time limit to plan,remember that they are 5 random people and so dont know each other,thus they need time agree and plan.

it's already possible to metagame within the first 5 minutes of a warops round based on the fact that they have 5 minutes to call for warops to begin with. i do not see how adding an auto-FTL to the shuttle after that would make it any different

**Comment by Ratonero500** [comment]
Yes you know that after the 5 minutes mark it cant be war ops if it hasnt be called,but what matters more is knowing when they will come,adding an autoftl will make the crew know when the ship arrives thus they can do things like search for it with salv even before the nukies get into station.

**Comment by retequizzle** [comment]
> Yes you know that after the 5 minutes mark it cant be war ops if it hasnt be called,but what matters more is knowing when they will come,adding an autoftl will make the crew know when the ship arrives thus they can do things like search for it with salv even before the nukies get into station.

which players can still already do once they know it's war ops, and the chances of people finding the ship after it FTLs but before it arrives at the station are so small and situational to begin with, and ALSO risks the salv ship in the process

keep in mind i'm not saying that the shuttle can ONLY ftl at 30 minutes, i'm only proposing a limit on how long nukies can spend on the nukie planet after issuing a war op declaration. if they stay on the nukie planet at that point, ship leaves without them, skill issue get rekt etc

**Comment by Ratonero500** [comment]
Ohhh that explains,yeah it makes more sense that way,but i think 30 minutes is quite low,i think something more akin to 45 or 50 minutes would be good.

**Comment by Brandon-Huu** [comment]
> > Yes you know that after the 5 minutes mark it cant be war ops if it hasnt be called,but what matters more is knowing when they will come,adding an autoftl will make the crew know when the ship arrives thus they can do things like search for it with salv even before the nukies get into station.
> 
> which players can still already do once they know it's war ops, and the chances of people finding the ship after it FTLs but before it arrives at the station are so small and situational to begin with, and ALSO risks the salv ship in the process
> 
> keep in mind i'm not saying that the shuttle can ONLY ftl at 30 minutes, i'm only proposing a limit on how long nukies can spend on the nukie planet after issuing a war op declaration. if they stay on the nukie planet at that point, ship leaves without them, skill issue get rekt etc

This happened twice to me without a forced FTL

**Comment by Brandon-Huu** [comment]
> > Yes you know that after the 5 minutes mark it cant be war ops if it hasnt be called,but what matters more is knowing when they will come,adding an autoftl will make the crew know when the ship arrives thus they can do things like search for it with salv even before the nukies get into station.
> 
> which players can still already do once they know it's war ops, and the chances of people finding the ship after it FTLs but before it arrives at the station are so small and situational to begin with, and ALSO risks the salv ship in the process
> 
> keep in mind i'm not saying that the shuttle can ONLY ftl at 30 minutes, i'm only proposing a limit on how long nukies can spend on the nukie planet after issuing a war op declaration. if they stay on the nukie planet at that point, ship leaves without them, skill issue get rekt etc

This is a role playing game. Forcing a FTL warp limits the role playing possibilities.

What if I want to play stealth ops with 30 nuclear operatives?
15 minutes it not enough time to get all of the players I need to do accomplish my plan.

**Comment by Brandon-Huu** [comment]
> > > > Yes you know that after the 5 minutes mark it cant be war ops if it hasnt be called,but what matters more is knowing when they will come,adding an autoftl will make the crew know when the ship arrives thus they can do things like search for it with salv even before the nukies get into station.
> > > 
> > > 
> > > which players can still already do once they know it's war ops, and the chances of people finding the ship after it FTLs but before it arrives at the station are so small and situational to begin with, and ALSO risks the salv ship in the process
> > > keep in mind i'm not saying that the shuttle can ONLY ftl at 30 minutes, i'm only proposing a limit on how long nukies can spend on the nukie planet after issuing a war op declaration. if they stay on the nukie planet at that point, ship leaves without them, skill issue get rekt etc
> > 
> > 
> > This is a role playing game. Forcing a FTL warp limits the role playing possibilities.
> > What if I want to play stealth ops with 30 nuclear operatives? 15 minutes it not enough time to get all of the players I need to accomplish my plan.
> 
> "stealth ops with 30 nuclear operatives"
> 
> that sounds like a clusterfuck of your own design and beyond the scope of issues i'm willing to consider with auto-FTL to be quite blunt with you

That sounds like fun you are trying to take away from me

**Comment by ketufaispikinut** [comment]
Closing as not planned.
